nb/o.n.upgrader/src/org/netbeans/upgrade/AutoUpgrade.java

```diff
@@ -133,73 +94,13 @@ private static boolean showUpgradeDialog (final File source, String note) {
 	JButton bNO = new JButton("No");
 	bNO.setMnemonic(KeyEvent.VK_N);
 	JButton[] options = new JButton[] {bYES, bNO};
-        JOptionPane p = new JOptionPane (panel, JOptionPane.QUESTION_MESSAGE, JOptionPane.YES_NO_OPTION, null, options, bYES);
-        JDialog d = Util.createJOptionProgressDialog(p, NbBundle.getMessage (AutoUpgrade.class, "MSG_Confirmation_Title"), source, progressBar);
+        JOptionPane p = new JOptionPane(panel, JOptionPane.QUESTION_MESSAGE, JOptionPane.YES_NO_OPTION, null, options, bYES);
+        JDialog d = Util.createJOptionProgressDialog(p, NbBundle.getMessage(AutoUpgrade.class, "MSG_Confirmation_Title"), source, progressBar);
         d.setVisible (true);
 
         return Integer.valueOf(JOptionPane.YES_OPTION).equals (p.getValue ());
     }
 
-    private static void showNoteDialog (String note) {
-        Util.setDefaultLookAndFeel();
-        JOptionPane p = new JOptionPane(new AutoUpgradePanel (null, note), JOptionPane.INFORMATION_MESSAGE, JOptionPane.DEFAULT_OPTION);
-        JDialog d = Util.createJOptionDialog(p, NbBundle.getMessage (AutoUpgrade.class, "MSG_Note_Title"));
-        d.setVisible (true);
-    }
-
-    static void doUpgrade (File source, String oldVersion) 
```

Comment:
this method has a test but was not used at runtime. I moved a simpler version of this method into the `AutoUpgradeTest` so that no test coverage of `Copy.copyDeep` is lost.

Reply:
I moved the test from AutoUpgrade to CopyTest.